README.md

```diff
@@ -39,10 +40,23 @@ jupyter-devinstall jdfreder ~/
 
 ## Notes
 
-If you have specified the `--silent` (`-s` for short) flag, part way through the
-tool will behave like a wizard, prompting you for input.  
+If you not have specified the `--silent` (`-s` for short) flag, part way through 
+the tool will behave like a wizard, prompting you for input.  
 
 The tool will ask you if you want to install locally, globally, or not at all:  
 - locally - pip install each repository  
 - globally - pip install each repository with the `-g` flag.  This may require sudo, which may have undesired side effects.  
 - not at all - doesn't pip/npm install anything  
+
+## Default workflow
+
+The default workflow installed by this tool is that origin points to your fork
+of the upstream repository.  You can change this behavior by using the `-U` flag,
+to force origin to point to the upstream repo.  For example,
+
+```
+jupyter-devinstall jdfreder ~/ -U -u me
```

Comment:
In my case then, since I keep things Go-style:

```
kyle@carina ~/code/src/github.com/jupyter$ ls
atom-notebook          jupyter-js-services    jupyter_console        nbviewer               tmpnb
enhancement-proposals  jupyter.github.io      jupyter_core           notebook               tmpnb-deploy
jupyter-js-output-area jupyter_client         nbformat               oauthenticator
```

I would then just `jupyter-devinstall rgbkrk ~/code/src/github.com/jupyter` I take it?